deploy/deploy/aws/config.py

```diff
@@ -0,0 +1,29 @@
+from common.config.config import CentralityConfig
+from typing import Optional
+from pydantic import model_validator
+
+
+class AwsDeployConfig(CentralityConfig):
```

Comment:
Add class and function comments to explain why some must be set

Reply:
Remember, centrality config always needs a default value, which is why we are doing this empty string crap